tests/convert/test_wei.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/python3
+import pytest
 
+from brownie import Fixed
```

Comment:
probably better to import both `Fixed` and `Wei` from where they are actually declared, `brownie.convert.datatypes`